package.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <package>
   <name>rosjava_core</name>
-  <version>0.3.0</version>
+  <version>0.3.1</version>
```

Comment:
thanks for this suggestion, but please leave it out of the update, this is done as part of the release process, it is better handled on a separate commit.